retriever/lib/engine.py

```diff
@@ -425,18 +429,20 @@ def download_file(self, url, filename):
                            unit_divisor=1024,
                            miniters=1,
                            desc='Downloading {}'.format(filename))
+            
             try:
+                requests.get(url, allow_redirects=True, stream=True,
```

Comment:
There are some indentation issues that need to be fixed here. 
And Am wondering if 
`headers={'user-agent': 'Weecology/Data-Retriever'},`
 the user agent could be more like

`https://github.com/weecology/retriever`
or 
`http://www.data-retriever.org/`
Something more specific. 
Disclaimer: I don't know the constraints or form of values that we can provide.